torch/csrc/distributed/rpc/python_rpc_handler.cpp

```diff
@@ -22,8 +22,8 @@ PythonRpcHandler::PythonRpcHandler() {
   AutoGIL ag;
   py::object module =
       py::module::import("torch.distributed.internal_rpc_utils");
-  runUDFFunction_ = getFunction(module, "run_python_udf_internal");
-  loadResultFunction_ = getFunction(module, "load_python_udf_result_internal");
+  pythonUDFRun_ = getFunction(module, "_python_udf_run");
```

Comment:
Do you think "Python" + "UDF" is a good name here?

It's just any Python function.


I learned the word UDF in SQL, SparkSQL, where UDF is somoething beyond the built-in capability of the domain-specific language.

Reply:
Good point. I'll use @mrshenli's suggestion below.